daft/daft/__init__.pyi

```diff
@@ -870,6 +870,7 @@ def read_parquet_into_pyarrow(
     io_config: IOConfig | None = None,
     multithreaded_io: bool | None = None,
     coerce_int96_timestamp_unit: PyTimeUnit | None = None,
+    coerce_string_to_binary: bool | None = None,
```

Comment:
we should also document what this is and why one would want to use it (such as reading non-valid utf8)

Reply:
In which files would the docs go into?

Reply:
I think the public [daft.read_parquet](https://github.com/universalmind303/Daft/blob/43965fe8949bab9e3a4d5f996fb1116f2f8efc1a/daft/io/_parquet.py#L21) has all of the documentation for other arguments. We should probably propagate this argument up to there as well

Reply:
Will add `string_encoding` to the actual `read_parquet` function in a follow up. For now, leaving it undocumented for this internal API should suffice.